src/main/resources/lib/unsplash/client.ts

```diff
@@ -0,0 +1,53 @@
+import { request } from "/lib/http-client";
+import { SearchPhotosParams, UnsplashResponseBody } from "/lib/unsplash/types";
```

Comment:
Using the `type` hint may improve performance.

```suggestion
import type { SearchPhotosParams, UnsplashResponseBody } from "/lib/unsplash/types";
```